code/game/gamemodes/objectives/basic/steal.dm

```diff
@@ -38,11 +38,21 @@ GLOBAL_LIST_EMPTY(possible_items)
 		explanation_text = "Steal [targetinfo.name]"
 		if (length(targetinfo.special_equipment))
 			generate_stash(targetinfo.special_equipment)
+		update_explanation_text()
 		return steal_target
 	else
 		explanation_text = "Free objective"
 		return
 
+/datum/objective/steal/update_explanation_text()
+	if (!targetinfo)
+		explanation_text = "Free objective"
+		return
+	explanation_text = "Steal [targetinfo.name]"
```

Comment:
no dot.

Reply:
Punctuation is inside of the name of the steal item

Reply:
oh... this was tricky